### ISSUE TITLE
Can't disable for a block

### ISSUE THREAD

**Issue by jtiscione** [eslint/eslint#10333]
```
// eslint-disable no-unused-vars
const x = 0;
// eslint-enable no-unused-vars
```
It still complains about x not being used.

**Comment by platinumazure** [comment]
Hi @jtiscione, thanks for the issue!

The `/* eslint-disable */` and `/* eslint-enable */` directives must be placed inside block comments. This should work for you:

```js
/* eslint-disable no-unused-vars */
const x = 0;
/* eslint-enable no-unused-vars */
```

**Comment by not-an-aardvark** [comment]
Closing because the question has been answered.
